docs/advanced-application/quiz-toolkit.md

```diff
@@ -0,0 +1,189 @@
+---
+sidebar_label: "Quiz Toolkit"
+sidebar_position: 1
+---
+# Quiz Toolkit
+
+| :trophy: This is a [premium](https://mecsimcalc.com/pricing) feature |
+| -------------------------------------------------------------------- |
+
+**MecSimCalc** now supports online quizzes.
+In this tutorial, we'll learn to build a Quiz App for educational purposes. The app displays scores 
+instantly, saves results to **Google Sheets**, and **sends email** to receivers(professors), streamlining 
+feedback and record-keeping.
+
+Seamlessly integrates with **[Google Sheets](google-sheet)** for straightforward and organized data compilation.
+![img alt](/docs/advanced-application/quiz-toolkit/sheet_res.png)
```

Comment:
For these initial google sheets & gmail images can we use the same ones as further down in the tutorial (to match the code)